### DISCUSSION TITLE
Raspotify skipping/not playing new releases

### DISCUSSION THREAD

**Discussion by djduckdj** [dtcooper/raspotify#631] [Q&A]
Hi!
I recently got raspotify set up on a pi 3b (as a user service following the setup guide as I'm outputting to a bluetooth speaker https://github.com/dtcooper/raspotify/wiki/Play-via-Bluetooth-Speaker). It's been working perfectly until today I noticed when trying to play a newly released song (released 13 Jun 2023) it refuses to play the song and will skip to the next song. The song will play fine on other devices (phone, laptop).

Has anyone had this issue or anything similar?

Thanks! :)

**Comment by JasonLG1979** [comment]
Logs?

**Reply by djduckdj** [reply]
Sorry I'm pretty new to all this, I've had a look at the logs using `sudo journalctl -u raspotify -b` but the only events in the logs are `Started Raspotify`, `Stopping Raspotify` and `raspotify.service: Succeeded.`. There's also nothing at `/etc/raspotify/crash_report`.

**Reply by JasonLG1979** [reply]
What is the song? It may be an incompatible format. Librespot only works with ogg vorbis streams. 99.99% of Spotify music is ogg vorbis, but every once in a while they throw out some random other audio format.

**Reply by djduckdj** [reply]
Ah I see, the song is High Tones by Cassia. How can you tell which format songs use?

**Reply by JasonLG1979** [reply]
You don't really. If it plays on librespot it's a ogg vorbis. If it doesn't it's more than likely not.

**Reply by JasonLG1979** [reply]
Podcasts have the same problem

**Reply by djduckdj** [reply]
Thanks for the response and sorry for not replying sooner, I just tried to play the same song again and it seems to play fine now!
Not sure exactly why it wasn't working but thanks for your time!

**Reply by JasonLG1979** [reply]
Maybe in the meantime the file was converted to Ogg Vorbis? The internals of the Spotify API are opaque. There is no way to know what happens behind the curtains.
